app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
@lsitu It didn't say the jpg file is corrupted. 
It just say change Content-Type from application/octet-stream to image/jpeg.
The jpg file itself doesn't have MIME type. But it could set up in the damspas program, which it was set as application/octet-stream in else block.